pkg/docker/config/config.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/containers/image/v5/pkg/sysregistriesv2"
 	"github.com/containers/image/v5/types"
 	"github.com/containers/storage/pkg/homedir"
+	"github.com/docker/distribution/reference"
```

Comment:
This should  be `github.com/containers/image/v5/docker/reference`.  I guess your IDE auto-imported this one.

Reply:
Thank you for the hint, fixed as suggested :+1: